app/controllers/report_controller/widgets.rb

```diff
@@ -244,7 +244,7 @@ def get_all_widgets(nodeid = nil, rep_id = nil)
     if x_active_tree == :widgets_tree
       # dont need to set these for report show screen
       @right_cell_div     = "widget_list"
-      @right_cell_text ||= _("All %s") % "MiqWidget"
+      @right_cell_text ||= _("All MiqWidgets")
```

Comment:
I know this was this way also before this change, but I think `MiqWidgets` (model name) should
be modified to `Widgets`.